lib/timers.js

```diff
@@ -23,13 +23,18 @@
 
 const TimerWrap = process.binding('timer_wrap').Timer;
 const L = require('internal/linkedlist');
+const timers = require('internal/timers');
```

Comment:
Grepping for `internal[A-Z]\w+ = require` in `lib/` gives 24 results, so I think it’s safe to say this is convention. :) I just noticed because at first I thought this refers to the `timers` module, and then I was confused because this *is* the source code for that module, and… you get the idea.